README.md

```diff
@@ -0,0 +1,17 @@
+[![Build Status Travis](https://travis-ci.org/RGF-team/rgf.svg?branch=master)](https://travis-ci.org/RGF-team/rgf)
+[![Build Status AppVeyor](https://ci.appveyor.com/api/projects/status/u3612bfh9pmela42/branch/master?svg=true)](https://ci.appveyor.com/project/RGF-team/rgf)
+[![Python Versions](https://img.shields.io/pypi/pyversions/rgf_python.svg)](https://pypi.org/project/rgf_python)
+[![PyPI Version](https://badge.fury.io/py/rgf_python.svg)](https://badge.fury.io/py/rgf_python)
+
+# Regularized Greedy Forest
+
+Regularized Greedy Forest (RGF) is a tree ensemble machine learning method described in [this paper](https://arxiv.org/abs/1109.0887).
+RGF can deliver better results than gradient boosting decision tree (GBDT) on a number of datasets and it have been used to win some Kaggle competitions.
+Unlike the traditional boosted decision tree approach, RGF directly works with the underlying forest structure.
+RGF integrates two ideas: one is to include tree-structured regularization into the learning formulation; and the other is to employ the fully-corrective regularized greedy algorithm.
```

Comment:
We'd better revise this expression by Rie Johnson or Tong Zhang.

Reply:
This description is taken from the original paper, so no need to do this.